### PR TITLE
Doc-84 RP doesn't support accessing managed deployments via kubectl

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://redpandadata.atlassian.net/browse/DOC-84
Review deadline: Dec 19

## Page previews
https://deploy-preview-159--rp-cloud.netlify.app/redpanda-cloud/get-started/cloud-overview/#byoc-architecture

The same note is included at the bottom of each BYOC page.

## Checks

- [ ] New feature
- [X] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

**Review by vuldin — APPROVED** [review]

File: modules/get-started/pages/cluster-types/byoc/azure/vnet-azure.adoc [hunk 1]
```diff
@@ -435,22 +435,7 @@ Example retrieving cluster:
...
-== Delete cluster
-
-To delete the cluster, first send a DELETE request to the Cloud API, and retrieve the `resource_id` of the DELETE operation. Then run the `rpk` command to destroy the cluster identified by the `resource_id`.
-
-```bash
-export REDPANDA_ID=$(curl -X DELETE "https://api.redpanda.com/v1beta2/clusters/${REDPANDA_ID}" \
- -H "accept: application/json"\
- -H "content-type: application/json" \
- -H "authorization: Bearer ${BEARER_TOKEN}" | jq -r '.operation.resource_id')
+ -H "authorization: Bearer $YOUR_TOKEN"
```
Comment (david-yu) [inline modules/get-started/pages/cluster-types/byoc/azure/vnet-azure.adoc:438]:
```suggestion
 -H "authorization: Bearer ${BEARER_TOKEN}"
```

File: modules/get-started/pages/cluster-types/byoc/azure/vnet-azure.adoc [hunk 2]
```diff
@@ -438,19 +438,4 @@ curl -X GET "https://api.redpanda.com/v1beta2/clusters/<resource_id of cluster f
  -H "authorization: Bearer ${BEARER_TOKEN}"
 ```
 
-== Delete cluster
```
Comment (david-yu) [inline modules/get-started/pages/cluster-types/byoc/azure/vnet-azure.adoc:441]: Is there a reason why delete cluster is removed? This should still be here in my opinion.

Comment (micheleRP) [reply modules/get-started/pages/cluster-types/byoc/azure/vnet-azure.adoc:441]: gah, yes, thank you! That was a merge mistake. It's back in now.

File: modules/get-started/partials/no-access.adoc [hunk 3]
```diff
@@ -0,0 +1 @@
+NOTE: To manage all configuration changes for a 99.99% service level agreement (SLA) on BYOC, Redpanda does not support user access to the control plane with `kubectl`.
```
Comment (JakeSCahill) [inline modules/get-started/partials/no-access.adoc:1]:
```suggestion
NOTE: Redpanda Cloud does not support user access to the control plane with `kubectl`. This restriction allows Redpanda Data to manage all configuration changes internally to ensure a 99.99% service level agreement (SLA) for BYOC deployments.
```

**Review by JakeSCahill — APPROVED** [review]
One suggestion to frontload the sentence that means the most to users.

**Pull request merged**